Z2A UIKit/NSCoffee/NSCoffee/Models/Basket.swift

```diff
@@ -5,16 +5,32 @@
 //  Created by Dani on 01/03/2025.
 //
 
+typealias Extras = [(description: String, price: Double)]
```

Comment:
What are you intending to use the description text for here?

Reply:
Just to show something that indicates you've added some extras, like extra shots, in the basket. For example, for "Espresso", it would show in the subtitle something like: "1 x £2.00 + (3 shots £1.20) = £3.20", where "3 shots" is the description.